controllers/storagecluster/exporter.go

```diff
@@ -759,6 +760,12 @@ func updateMetricsExporterClusterRoles(ctx context.Context, r *StorageClusterRec
 				Resources: []string{"objectbuckets"},
 				Verbs:     []string{"get", "list"},
 			},
+			{
+				APIGroups:     []string{"security.openshift.io"},
+				Resources:     []string{"securitycontextconstraints"},
+				ResourceNames: []string{"hostnetwork-v2"},
+				Verbs:         []string{"use"},
+			},
```

Comment:
We shouldn't be touching (creating/updating) RBAC from code. Why is this needed? why can't we just add these statically via olm?

Reply:
The changes are here to align with the already existing flow for (creating/updating) RBAC for metrics exporter,
If the changes here doesn't make sense then will add static changes

Reply:
@rohan47 The fact that we have some code that stray form the boundaries of what the operator should be able to do or shouldn't be able to do, which we also consider a mistake and a problem. Does not mean we should add to that problem be adding more

Please use a build-time/static approach for the additional rbac

Reply:
Made changes